docs/source/df.rst

```diff
@@ -0,0 +1,203 @@
+.. default-domain:: cpp
```

Comment:
- Place this file in this folder: `docs/source/onedal/algorithms/ensembles` (you'd need to create it).
- Rename the file as `decision-forest.rst`  
- Add the following at the very beginning:

```
.. ******************************************************************************
.. * Copyright 2020 Intel Corporation
.. *
.. * Licensed under the Apache License, Version 2.0 (the "License");
.. * you may not use this file except in compliance with the License.
.. * You may obtain a copy of the License at
.. *
.. *     http://www.apache.org/licenses/LICENSE-2.0
.. *
.. * Unless required by applicable law or agreed to in writing, software
.. * distributed under the License is distributed on an "AS IS" BASIS,
.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
.. * See the License for the specific language governing permissions and
.. * limitations under the License.
.. *******************************************************************************/

```

Reply:
applied